smol_podcaster.py

```diff
@@ -202,6 +233,12 @@ def main(url, name, speakers_count):
     
     print('Starting transcription')
     
+    # check if url is file with os, upload to tmpfiles if is
+    if os.path.exists(url):
+        url = upload_file_and_use_url(url)
+        # convert url by adding dl/ after https://tmpfiles.org/
+        url = url.replace("https://tmpfiles.org/", "https://tmpfiles.org/dl/")
```

Comment:
I'd probably move this in the `upload_file_and_use_url`, if in the future we change tmp to something else this code will also need to change so better to incapsulate it together?